rust-runtime/aws-smithy-runtime/Cargo.toml

```diff
@@ -53,7 +53,8 @@ pretty_assertions = "1.4.0"
 tokio = { version = "1.25", features = ["macros", "rt", "rt-multi-thread", "test-util", "full"] }
 tracing-subscriber = { version = "0.3.16", features = ["env-filter"] }
 tracing-test = "0.2.1"
-hyper_0_14 = { package = "hyper", version = "0.14.27",features = ["client", "server", "tcp", "http1", "http2"] }
+hyper_0_14 = { package = "hyper", version = "0.14.27", features = ["client", "server", "tcp", "http1", "http2"] }
+http1 = { package = "http", version = "1" }
```

Comment:
Might want to name this dependency something like `http1x` to make it every so slightly less likely to be confused with the HTTP 1 vs. HTTP 2 protocol versions, although that may not make it much better 🤔